backend/src/Response/Admin/StoreOwnerSubscription/AdminStoreSubscriptionResponse.php

```diff
@@ -46,4 +46,18 @@ class AdminStoreSubscriptionResponse
      *     @OA\Items(type="object"))
      */
     public $captainOffers;
+
+    public int $packageCarCount;
+
+    public int $packageOrderCount;
+
+    public int $packageExpired;
+
+    public string $packageNote;
```

Comment:
package note may be null, so we have to set null as probably return type for packageNote beside string.